src/main/cli/settings.c

```diff
@@ -1133,6 +1133,7 @@ const clivalue_t valueTable[] = {
 #ifdef USE_ADC_INTERNAL
     { "osd_core_temp_alarm",        VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, UINT8_MAX }, PG_OSD_CONFIG, offsetof(osdConfig_t, core_temp_alarm) },
 #endif
+    { "osd_vbat_digits",            VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 2, 4 }, PG_OSD_CONFIG, offsetof(osdConfig_t, vbat_digits) },
```

Comment:
Making this configurable seems to be a bit over the top for me - probably reining in the number of significant digits that is used to 3 or so will be sufficient to address the concerns that led to this.

Reply:
Yes its another element no one will ever touch, i just tracked the original idea from the isue, so i will rework it for use 3 significant digits using current tfp_sprintf and if/when @ledvinap implement better printf it should be replaced by it.